client/core/core.go

```diff
@@ -4642,14 +4642,22 @@ func (c *Core) dbTrackers(dc *dexConnection) (map[order.OrderID]*trackedTrade, e
 				// tracker.cancel is set from LinkedOrder with cancelTrade.
 				continue
 			}
+			// Make sure that a taker will not prematurely send an
+			// initialization until it is confimed with the server
+			// that the match is not revoked.
+			var checkServerRevoke bool
+			if dbMatch.Side == order.Taker && dbMatch.Status == order.MakerSwapCast {
+				checkServerRevoke = true
+			}
```

Comment:
Should this also include the Maker with status TakerSwapCast to avoid redeeming if the match is revoked?

Reply:
Don't think so.  That's a good resolution to the swap IMO.  Late even if already penalized.  The other client can search for the redeem, but we have plans for server to accept and relay the redemption info _after revocation_.  I keep meaning to start that.

Reply:
ok.